app/controllers/service_requests_controller.rb

```diff
@@ -429,16 +429,20 @@ def setup_catalog_calendar
       end
     end
   end
-
+  
   def setup_catalog_news_feed
     if USE_NEWS_FEED
-      page = Nokogiri::HTML(open("https://www.sparcrequestblog.com"))
-      articles = page.css('article.post').take(3)
-      @news = []
-      articles.each do |article|
-        @news << {title: (article.at_css('.entry-title') ? article.at_css('.entry-title').text : ""),
+      begin
+        page = Nokogiri::HTML(open("https://www.sparcrequestblog.com"))
+        articles = page.css('article.post').take(3)
+        @news = []
+        articles.each do |article|
+          @news << {title: (article.at_css('.entry-title') ? article.at_css('.entry-title').text : ""),
                   link: (article.at_css('.entry-title a') ? article.at_css('.entry-title a')[:href] : ""),
                   date: (article.at_css('.date') ? article.at_css('.date').text : "") }
+        end
+      rescue Net::OpenTimeout
+        @news = []
```

Comment:
`@news = []` could be done before the `begin` block, that way there's no need to duplicate it in the `rescue` block